src/App.tsx

```diff
@@ -41,30 +42,37 @@ const App = () => {
       isRestored,
       isBackedUp,
       isRequestingPermission,
+      isLocked,
+      appStateStatus,
     },
   } = useSelector((state: RootState) => state)
 
+  useEffect(() => {
```

Comment:
Does this fix something? Just wondering. I see we moved from using the direct callback to using the state and effect instead. Makes sense just wondering if it caused a bug or something?

Reply:
There was a bug on android where face id would send the app into 'background' which was causing an infinite loop of lock=>auth=>lock=>auth

Reply:
Ah nice good find